doctr/models/predictor/base.py

```diff
@@ -64,12 +64,13 @@ def _process_predictions(
         word_preds: List[Tuple[str, float]],
     ) -> Tuple[List[np.ndarray], List[List[Tuple[str, float]]]]:
 
-        text_preds = []
+        boxes, text_preds = [], []
         if len(loc_preds) > 0:
             # Text
             _idx = 0
             for page_boxes in loc_preds:
+                boxes.append(page_boxes[:, :-1])
```

Comment:
Sorry I think I lost a bit the reason why we need to remove the last item on the 2nd axis, can you remind me?

Reply:
Boxes should either be of dimension 4 or 5. as far as I understood `loc_preds` have a probability parameter and can be of dimension 5 or 6. If you don't remove the last item on the 2nd axis, you are passing the rest of the code an array of boxes with wrond dimension. Downstream, there are many checks of the dimension of the array to verify is the boxes are straight or rotated. This gets confusing if you pass them the probability parameter.